exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -40,71 +44,66 @@ public string CreateRockQuestion(int index)
+        public int HowManyPlayersInCurrentGame()
         {
-            return players.Count;
+            return listOfAllPlayers.Count;
         }
 
-        public void Roll(int roll)
+        public void RollDice(int DiceRollResult)
         {
-            Console.WriteLine(players[currentPlayer] + " is the current player");
-            Console.WriteLine("They have rolled a " + roll);
+            Console.WriteLine(listOfAllPlayers[currentPlayer] + " is the current player");
+            Console.WriteLine("They have rolled a " + DiceRollResult);
 
             if (inPenaltyBox[currentPlayer])
             {
-                if (roll % 2 != 0)
+                if (DiceRollResult % 2 != 0)
                 {
                     isGettingOutOfPenaltyBox = true;
 
-                    Console.WriteLine(players[currentPlayer] + " is getting out of the penalty box");
-                    places[currentPlayer] = places[currentPlayer] + roll;
-                    if (places[currentPlayer] > 11) places[currentPlayer] = places[currentPlayer] - 12;
-
-                    Console.WriteLine(players[currentPlayer]
-                            + "'s new location is "
-                            + places[currentPlayer]);
-                    Console.WriteLine("The category is " + CurrentCategory());
-                    AskQuestion();
+                    Console.WriteLine(listOfAllPlayers[currentPlayer] + " is getting out of the penalty box");
+                    MovePlayerToNewPosition(DiceRollResult);
                 }
                 else
                 {
-                    Console.WriteLine(players[currentPlayer] + " is not getting out of the penalty box");
+                    Console.WriteLine(listOfAllPlayers[currentPlayer] + " is not getting out of the penalty box");
                     isGettingOutOfPenaltyBox = false;
                 }
 
             }
             else
             {
-
-                places[currentPlayer] = places[currentPlayer] + roll;
-                if (places[currentPlayer] > 11) places[currentPlayer] = places[currentPlayer] - 12;
-
-                Console.WriteLine(players[currentPlayer]
-                        + "'s new location is "
-                        + places[currentPlayer]);
-                Console.WriteLine("The category is " + CurrentCategory());
-                AskQuestion();
+                MovePlayerToNewPosition(DiceRollResult);
             }
 
         }
 
+        private void MovePlayerToNewPosition(int DiceRollResult)
+        {
+            playersPosition[currentPlayer] = playersPosition[currentPlayer] + DiceRollResult;
+            if (playersPosition[currentPlayer] > 11) playersPosition[currentPlayer] = playersPosition[currentPlayer] - 12;
+
+            Console.WriteLine(listOfAllPlayers[currentPlayer]
+                    + "'s new location is "
+                    + playersPosition[currentPlayer]);
+            Console.WriteLine("The category is " + CurrentCategory());
```

Comment:
This should be its own method. MovePlayerToNewPosition() should have one job: move the player. However, now it also has PrintUpdate()